app/scripts/modules/core/pipeline/config/triggers/travis/travisTriggerOptions.component.ts

```diff
@@ -65,17 +65,22 @@ export class TravisTriggerOptions {
   };
 }
 
+class TravisTriggerOptionsComponent implements IComponentOptions {
+  public bindings: any;
+  public templateUrl: string;
+  public controller: any;
+
+  constructor() {
+    this.bindings = {
+      command: '='
+    };
+    this.templateUrl = require('./travisTriggerOptions.component.html');
+    this.controller = TravisTriggerOptions;
+  }
+}
```

Comment:
Why isn't the recursive dependency a problem when using `templateUrl` and require?